static/js/compose.js

```diff
@@ -538,6 +544,13 @@ exports.transmit_message = function (request, success, error) {
         send_message_ajax(request, success, error);
     }
 };
+function report_content_not_sent(request) {
```

Comment:
We currently test if the message is delivered successfully. So this stubbing will have to be placed in another test function where we simulate the message not being delivered?